Assets/SteamVR_Unity_Toolkit/Scripts/Controls/2D/RadialMenu/Editor/RadialMenuInspector.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEditor;
+
+[CustomEditor(typeof(RadialMenu))]
+public class RadialMenuInspector : Editor
+{
+    public override void OnInspectorGUI()
+    {
+        DrawDefaultInspector();
+        
+		RadialMenu rMenu = (RadialMenu)target;
```

Comment:
Looks like there is a combination of tabs and spaces making the formatting go a bit crazy.

Probably worth auto formatting the code so it's consistent